source/beamline/PGM/PGMBeamline.cpp

```diff
@@ -67,6 +69,7 @@ void PGMBeamline::setupMono()
 
 void PGMBeamline::setupComponents()
 {
+    energy_ = new AMPVwStatusControl("Energy", "BL1611-ID-2:Energy:fbk", "BL1611-ID-2:Energy", "BL1611-ID-2:status", "BL1611-ID-2:emergStop", this, 0.001, 2.0, new CLSMAXvControlStatusChecker());
```

Comment:
@dleacock `BL1611-ID-2:emergStop` is not the stop PV for energy.

`BL1611-ID-2:emergStop` will try to stop a bunch of Beamline motors, including mono, PSL16114I2x01, SMTR16113I2010:emergStop, SMTR16113I2011:emergStop, SMTR1411-02:stop, FM16113I2001:emergStop, VAM16113I2001:emergStop PP, M16113I2001:emergStop PP, M16113I2002:emergStop, BL1611-ID-2:emergStop2.PROC.

I guess the  closest one is probably `PGM_mono:emergStop`, which will try to stop the mono motors: `SMTR16114I2001:emergStop`, `SMTR16114I2002:emergStop`, `SMTR16114I2003:emergStop` and `SMTR16114I2004:stop`